### ISSUE TITLE
Non-English language support

### ISSUE THREAD

**Issue by KasvanD** [alexkrechik/VSCucumberAutoComplete#297]
I may be overlooking something, but isn't it possible to have the auto complete functionality work in another language than English? I write my Feature Files in Dutch (Gegeven, Als Dan) instead of English (Given, When, Then). Is there a setting to have auto complete work in Dutch (or any other language supported by Cucumber)? If so how do I change the language setting? If not, can this functionality be added? 

Thanks.

**Comment by alexkrechik** [comment]
Hey!
Steps auto-complete should properly work for all the supported languages (including Dutch). Write some Dutch word (ex. Gegeven) and start to type step string after it - all the "Given" steps should appear.
Or do you mean "Gherkin" words snippets?

**Comment by KasvanD** [comment]
Yes, I indeed meant the Gherkin words snippets, sorry for the confusion.

**Comment by alexkrechik** [comment]
Got it. I'm not sure that I can add all the possible snippets due to there are hundreds of supported languages, but I'll look into this.

**Comment by KasvanD** [comment]
I only need Dutch :P haha...
Isn't it possible to retrieve the snippets directly from Cucumber, maybe? They're already written down there...  (didn't look into the code - so have no clue... just a thought that popped up!)

**Comment by tyler36** [comment]
Not sure if this is related but in Japanese the subject often comes first ...
```
Then( 'I type {string} into {string}', cy.typeStringIntoCkeditor );
もし( '{string}エディターに{string}と入力します', cy.typeStringIntoCkeditor );
```

How can I get the auto-complete to suggest all items that have a {string} as the first part?
